stable/prometheus/values.yaml

```diff
@@ -1,5 +1,7 @@
 rbac:
   create: true
+  podSecurityPolicy:
```

Comment:
I'm not sure enabling PSP is an RBAC related thing, so it probably doesn't belong inside the `rbac` structure here?